docs/application/dotnet/guides/nui/nui-components/RadioButton.md

```diff
@@ -0,0 +1,78 @@
+# RadioButton
+RadioButton is a common component and describes what action will occur when you select it.  
+A RadioButton can only contain an icon, and can be created using style.
+
+![RadioButton1](./media/RadioButton1.png)
+
+## Create with Style
+
+To create a radio button using style, follow these steps:
+
+1. Create RadioButton using the default constructor:
+
+    ```cs
+    RadioButton utilityRadioButton = new RadioButton();
+    ```
+
+2. Apply style to RadioButton:
+
+    ```cs
+    ButtonStyle utilityStyle = new ButtonStyle
+    {
+        Icon = new ImageViewStyle
+        {
+            Size =  new Size(48, 48),
+            ResourceUrl = new Selector<string>
+            {
+                Normal = "btn_radio_off.png",
+                Selected = "btn_radio_on.png",
+                Disabled = "btn_radio_off.png",
+                DisabledSelected = "btn_radio_on.png",
+            }
+        }
+    }
+    utilityRadioButton.ApplyStyle(utilityStyle);
+    utilityRadioButton.Size = new Size(48, 48);
+    root.Add(utilityRadioButton);
+    ```
+
+Following output is generated when a radio button is created by using style:
```

Comment:
@surbhig12 Minor question. Don't we need to remove "by" after "using"?

Following output is generated when a radio button is created by using style:

Reply:
@mijin85cho, I agree. I left this comment thinkign it is a minor commnt.
@zhouleonlei, request you to please incorporate thsi comment as well.

Following output is generated when a radio button is created using style:

Reply:
fixed